### ISSUE TITLE
Final peer review

### ISSUE THREAD

**Issue by sq89** [hya-cala/PredictH1BVisa#16]
If a foreign employee is hired by a company, he/she needs to be approved by LCA and wait for the H1B lottery. In this project, the 2010 SOC system dataset is used and set case_status as the response variable in the models. The team tries to predict whether the employee is approved by LCA in the H1B lottery with a decision tree, logistic regression, LDA & QDA, neural network, autogluon and lightGBM.
Things I like:
1. The paper is well organized and clear for the readers to follow the topic and methods that were used to approach the problem.
2. the group used many ways to fit the model and tried to find the best model among them. all the models were well explained in detail.
3. the group used many plots to show the results or the date and all of them were clear and easy to understand.

Things to improve:
1. There was no discussion about overfitting which is very important in the model fitting process. The group needs to talk about how they avoid overfitting while selecting the models.
2. Since the data set was imbalanced, accuracy may not be a good valuation metric in this project. The group may use other metrics to evaluate the models, like AUC, precision, recall, or others.
3. The group might add more to try to minimize the impact from the imbalanced data set, like using an oversampling or SMOTE.

In all. the group did a great job in this project and the paper is well structured and instructive.
